packages/ia-components/sandbox/write-review/button/button.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const button = props => {
```

Comment:
i think we need to take a step back and create a button component that is generic enough to be used in more ways than one. let's take a step back and look at the feature that you are working on, get @pezvi involved, and ensure that what you make is on brand and ready to use. 😄